ledger/acctupdates_test.go

```diff
@@ -71,11 +111,20 @@ func (ml *mockLedgerForTracker) Latest() basics.Round {
 	return basics.Round(len(ml.blocks)) - 1
 }
 
+func (ml *mockLedgerForTracker) addMockBlock(be blockEntry, delta StateDelta) error {
+	ml.blocks = append(ml.blocks, be)
+	ml.deltas = append(ml.deltas, delta)
+	return nil
+}
+
 func (ml *mockLedgerForTracker) trackerEvalVerified(blk bookkeeping.Block, accUpdatesLedger ledgerForEvaluator) (StateDelta, error) {
-	delta := StateDelta{
-		hdr: &bookkeeping.BlockHeader{},
+	if len(ml.deltas) > int(blk.Round()) {
+		return ml.deltas[uint64(blk.Round())], nil
+	} else {
+		return StateDelta{
+			hdr: &bookkeeping.BlockHeader{},
+		}, nil
```

Comment:
this looks not obvious at all, could you please comment/explain/make a separate method?